code/go/0chain.net/smartcontract/storagesc/block_reward.go

```diff
@@ -4,68 +4,103 @@ import (
 	cstate "0chain.net/chaincore/chain/state"
 	"0chain.net/chaincore/state"
 	"0chain.net/core/common"
+	"0chain.net/core/maths"
+	"0chain.net/smartcontract/partitions"
+	"math"
+	"math/rand"
+	"time"
 )
 
-func (ssc *StorageSmartContract) payBlobberBlockRewards(
+func (ssc *StorageSmartContract) blobberBlockRewards(
 	balances cstate.StateContextI,
 ) (err error) {
-	var conf *scConfig
+	var (
+		qualifyingBlobberIds []string
+		stakePools           []*stakePool
+		stakeTotals          []float64
+		totalQStake          float64
+		weight               []float64
+		totalWeight          float64
+		conf                 *scConfig
+	)
+
+	const (
+		// constants for gamma
+		alpha = 1
+		A     = 1
+		B     = 1
+
+		// constants for zeta
+		I  = 1
+		K  = 1
+		mu = 1
```

Comment:
These six constants need to be added to smart contract settings read from `sc.yaml` Check slack thread. For six parameters it might be worth creating a separate object that can be passed to the maths package.

Reply:
Maybe add a TODO comment or create an issue to track this

Reply:
added a TODO and created an issue: https://github.com/0chain/0chain/issues/1035